docker/ubuntu-based/prod/vswitch/vppctl

```diff
@@ -13,12 +21,28 @@ then
     exit 1
 fi
 
-# There may be more than one vswitch container if an upgrade/redeploy is happening, so go for the newest one.
-if ! ID=$(docker ps --format '{{.CreatedAt}} {{.Image}} {{.ID}}' | grep -E " $IMAGE@?[^ ]* [^ ]+$" | sort -nr | \
-    head -n 1 | sed -e 's|.* ||g') || [ -z "$ID" ]
-then
-    echo "Error finding a running container derived from the $IMAGE image." >&2
-    exit 1
+ID=$(get_vswitch_container_ID "$IMAGE")
```

Comment:
If this command fails, it will kill the script, which is probably not what you want.

Reply:
See above - where should the test for unbounded variable be put - inside the function or everywhere where the function is used?